### PR TITLE
Adapt expose tutorial and add multi namespace one

### PR DESCRIPTION
**Pull request by videlov**
**Description**

Added optional `namespace` in `service` for APIRule CRD (on root `spec` or as subelement in `rules`)

Changes in this pull request:

- [x] Update API-Gateway exposure tutorial
- [x] Add new exposure tutorial for multiple namespace services

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/9936
https://github.com/kyma-project/api-gateway/issues/12

### CODE REVIEW

File: docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+1. Create a Namespace for the Gateway and APIRule CRs. Skip the step if you already have a Namespace. Run:
+
+   ```bash
+   export NAMESPACE={NAMESPACE}
+   kubectl create ns $NAMESPACE
+   kubectl label namespace $NAMESPACE istio-injection=enabled --overwrite
+   ```
+
+2. Export the following value as an environment variable:
```
Comment (werdes72) [inline docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md:52]:
```suggestion
2. Export the following values as an environment variables:
```

File: docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+4. Create a Function using the [supplied code](./assets/function.yaml) in its Namespace:
+
+   ```bash
+   kubectl -n $NAMESPACE_FUNCTION apply -f https://raw.githubusercontent.com/kyma-project/kyma/main/docs/03-tutorials/assets/function.yaml
+   ```
+
+## Next steps
+
+1. Create a Namespace for the Gateway and APIRule CRs. Skip the step if you already have a Namespace. Run:
```
Comment (werdes72) [inline docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md:44]:
```suggestion
1. Create a Namespace for the Gateway and APIRule CRs. Skip this step if you already have a Namespace. Run:
```

File: docs/03-tutorials/00-api-exposure/apix-03-expose-workload-apigateway.md [hunk 3]
```diff
@@ -117,6 +121,8 @@ Follow the instruction to expose and access your unsecured instance of the HttpB
 
    >**NOTE:** If you are running Kyma on k3d, add `httpbin.kyma.local` to the entry with k3d IP in your system's `/etc/hosts` file.
 
+   >**NOTE:** You may specify a Namespace for your service. If you omit it, the default is then the APIRule Namespace.
```
Comment (SamehMKhattab) [inline docs/03-tutorials/00-api-exposure/apix-03-expose-workload-apigateway.md:124]:
```suggestion
   >**NOTE:** If you didn't specify a Namespace for your service, the default APIRule Namespace will be used.
```

File: docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+---
+title: Expose a workloads in multiple namespaces
+---
+
+This tutorial shows how to expose service endpoints in multiple namespaces using API Gateway Controller.
+   > **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always secure the workload you expose with [OAuth2](./apix-05-expose-and-secure-workload-oauth2.md) or [JWT](./apix-08-expose-and-secure-workload-jwt.md).
+
+The tutorial comes with a sample HttpBin and a sample Function service deployments.
```
Comment (SamehMKhattab) [inline docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md:8]:
```suggestion
```

File: docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+2. Export the following values as environment variables:
+
+   ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE_APIRULE/httpbin-gateway
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com. If you don't want to use your custom domain, replace `DOMAIN_NAME` with a Kyma domain and `$NAMESPACE/httpbin-gateway` with Kyma's default Gateway `kyma-system/kyma-gateway`
+
+3. Expose the HttpBin service and the Function service by creating an APIRule CR in their respective Namespaces. Run:
```
Comment (strekm) [inline docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md:61]: APIRule is created in its own namespace

**Review by strekm — DISMISSED** [review]
/meow

File: docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md (hunk 5 above)
Comment (videlov) [reply docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md]: Please see if it is now more understandable.

**Review by strekm — APPROVED** [review]

File: docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+2. Export the following values as environment variables:
+
+   ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE_APIRULE/httpbin-gateway
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com. If you don't want to use your custom domain, replace `DOMAIN_NAME` with a Kyma domain and `$NAMESPACE/httpbin-gateway` with Kyma's default Gateway `kyma-system/kyma-gateway`
+
+3. Expose the HttpBin and Function services in their respective Namespaces by creating an APIRule CR which is in its own namespace. Run:
```
Comment (grego952) [inline docs/03-tutorials/00-api-exposure/apix-09-expose-workloads-multiple-namespaces.md:61]:
```suggestion
3. Expose the HttpBin and Function services in their respective Namespaces by creating an APIRule CR which is in its own Namespace. Run:
```

**Review by grego952 — APPROVED** [review]

**Pull request merged**